test/smoke/src/helpers/smokeTestsConstants.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+export class smokeTestsConstants {
+// Android 9 = Pie
+public static standardAndroidPlatformVersion = "9";
+// Standard emulator name "emulator-"+port
+public static standardAndroidDeviceName = "emulator-5554";
+// Allow ten minutes for setup
+public static smokeTestSetupAwaitTime = 10 * 60 * 1000;
+// Await for emulator boot
+public static emulatorLoadTime = 90 * 1000;
+// Time that Spectron will wait for UI elements until it throw an Error
+public static spectronElementWaitTime = 150 * 1000;
+// Time for enabling Remote JS Debugging while testing RN app
+public static enableRemoteJSWaitTime = 30 * 1000;
+// Time for Android app build and install
+public static timeForAndroidAppBuildAndInstall = 100 * 1000;
+// Time for Android Debug Test before it reach timeout
+public static debugAndroidTestTime = 200 * 1000;
```

Comment:
I'd left it inside the test class because this constant will be used only there.